### PR TITLE
Docs - docker & man page

### PR DESCRIPTION
**Pull request by chmac**
Added some extra docs, with a couple of links from the readme. Feel free to edit, just a starting suggestion.

### CODE REVIEW

**Comment by jelmer** [comment]
Thanks for the PR. There's already a manual page in man/xandikos.8

**Comment by chmac** [comment]
Yeah, I converted it into a markdown file so it's readable here on GitHub.

I just piped it into:

    pandoc -f man -t markdown

Out came markdown. The man file directly is not really readable on GitHub.

**Comment by jelmer** [comment]
I don't really want to carry two copies of the same file in the repository - they'll get out of sync. I'm open to the idea of maintaining the manpage in markdown, but in that case please set up the appropriate tooling to generate it with the Makefile.

**Comment by chmac** [comment]
@jelmer Makes sense, I can understand that. I'm not so familiar with makefiles. I think linking to the ubuntu hosted man page makes a lot of sense. The only slight risk is if they get out of sync, but that seems like a small issue.

**Comment by jelmer** [comment]
If you think it's important to have the manpage accessible on the web, perhaps we can just publish it as part of the xandikos homepage?

**Comment by chmac** [comment]
@jelmer I think the man page already being online is totally fine, honestly I simply never thought of searching for it, I didn't realise xandikos was distributed. I also didn't think about the man page at all as I don't usually think about running commands inside docker containers.

I've reduced the changes to just a link and the example compose file. I think that'll point future docker users in the right direction along with #145.

Thanks for being so receptive to all this input by the way, it's not every author who's as open to feedback, I really appreciate it.

File: README.rst [hunk 1]
```diff
@@ -84,7 +84,10 @@ Docker
 
 A Dockerfile is also provided; see the comments on the top of the file for
 configuration instructions. The docker image is regularly built and
-published at ``ghcr.io/jelmer/xandikos``.
+published at ``ghcr.io/jelmer/xandikos``. See 
+``examples/docker-compose.yml`` and the 
+`man page <http://manpages.ubuntu.com/manpages/impish/man1/xandikos.1.html>`_
```
Comment (jelmer) [inline README.rst:89]: I'd rather not link to an external site - especially as this link is tied to a particular ubuntu version. Let's publish the manpage on the xandikos site and then link to that. I'd be great if you could submit a PR to https://github.com/xandikos/homepage but I'm also happy to take a look at that (I think it's fairly easy).

Comment (jelmer) [reply README.rst:89]: I've added a copy on https://www.xandikos.org/manpage.html (may take up to 30 minutes to propagate)

Comment (chmac) [reply README.rst]: Awesome, I've updated the PR to use that url instead. 👍

**Comment by chmac** [comment]
@jelmer I notice this was approved but not merged yet. Was there a reason to wait to merge?

**Pull request merged**